using_images/db_images/mongodb.adoc

```diff
@@ -456,54 +456,76 @@ of the *rs0* replication set, as the value of `*MONGODB_REPLICA_NAME*` is set to
 === Creating the Service Pod
 
 To initialize members created by xref:creating-the-deploymentconfig[the
-deployment configuration], a _service pod_ is defined in the template. This pod
-starts MongoDB with the `initiate` argument, which instructs the container
-entrypoint to behave xref:mongo-creating-headless-service[slightly differently]
+deployment configuration], the pods are started with the `initiate` argument, 
+which instructs the container entrypoint to behave xref:mongo-creating-headless-service[slightly differently]
 than a regular, stand-alone MongoDB database.
 
-[[example-service-pod]]
-.Service Pod Object Definition in the Example Template
+[[example-deployment-config]]
+.Deployment Configuration Object Definition in the Example Template
 ====
 
 [source,yaml]
 ----
-kind: "Pod"
-apiVersion: "v1"
-metadata:
-  name: "mongodb-service"
-  creationTimestamp: null
-  labels:
-    name: "mongodb-service"
-spec:
-  restartPolicy: "Never"
-  dnsPolicy: "ClusterFirst"
-  containers:
-    - name: "initiate"
-      image: "centos/mongodb-26-centos7"
-      args:
-        - "initiate"
-      env:
-        - name: "MONGODB_USER"
-          value: "${MONGODB_USER}"
-        - name: "MONGODB_PASSWORD"
-          value: "${MONGODB_PASSWORD}"
-        - name: "MONGODB_DATABASE"
-          value: "${MONGODB_DATABASE}"
-        - name: "MONGODB_ADMIN_PASSWORD"
-          value: "${MONGODB_ADMIN_PASSWORD}"
-        - name: "MONGODB_REPLICA_NAME"
-          value: "${MONGODB_REPLICA_NAME}"
-        - name: "MONGODB_SERVICE_NAME"
-          value: "${MONGODB_SERVICE_NAME}"
-        - name: "MONGODB_KEYFILE_VALUE"
-          value: "${MONGODB_KEYFILE_VALUE}"
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: "${MONGODB_SERVICE_NAME}"
+  spec:
+    strategy:
+      type: Recreate
+      recreateParams:
+        post:
+          failurePolicy: Retry
+          execNewPod:
+            command:
+            - run-mongod
+            - initiate
```

Comment:
incidentally the format you get back from "oc get foo -o yaml" is as it's represented in this PR as well, so i can change it, but it's non-standard as far as go seems to be concerned.